### PR TITLE
the desktop version project

### PR DESCRIPTION
**Pull request by basem909**
- this part of the portfolio project aims to turn the website into a responsive one that suits most of the screens 

- media queries added to the CSS style sheet to make the page responsive 

- some changes were made on the HTML file as well to fit the new approach to the responsive site.

- this part of the project is a solo coding part.

- languages used are HTML and CSS some functionality needs to be done through JS but it is not required at this level.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,58 +22,81 @@
...
-        <section class="project">
-          <div>
+      <section class="project">
+        <div class="back1"></div>
+        <div class="back2"></div>
+        <div class="back3"></div>
+        <div class="back4"></div>
+
+        <article class="projects">
+          <h2 class="header-project">projects</h2>
```
Comment (thecodechaser) [inline index.html:57]: - [ ] Good job on the styles but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)template as much as possible. Please use the correct font-weight for your `projects` and `about` headings and make sure they match the design on the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9).

### Your design
image

### Figma design
image

File: index.html [hunk 2]
```diff
@@ -223,22 +266,43 @@ <h3 class="skills-header">Skills</h3>
...
       <!--the third part of the project-->
       <section class="contact">
-        <h3 class="contact-header">Get started</h3>
-        <p class="contact-paragraph">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna.
-        </p>
+        <article class="introduction-contact">
+          <h3 class="contact-header">Get started</h3>
+          <p class="contact-paragraph">
```
Comment (thecodechaser) [inline index.html:293]: - [ ] Please make sure this section is in the center as on the design.

### Your design
image

### Figma design
image

File: index.html [hunk 3]
```diff
@@ -174,26 +226,17 @@ <h3 class="name-project">project name goes here</h3>
...
-                alt="get my cv icon"
-              />
-            </span>
-          </button>
+        </article>
+      </section>
+      <section class="about-package">
+        <article class="package">
           <section class="languages">
             <h3 class="lang-header">languages</h3>
```
Comment (thecodechaser) [inline index.html:240]: - [ ] Please place some left, right margins for this section to match the design.

### Your design

image

### Figma design

image

File: index.html [hunk 4]
```diff
@@ -22,58 +22,81 @@
...
-          Hello! I am a software developer! I can help you build a product,
-          feature or website. Take a look of my works. If you like what you see
-          and have a project you need coded, do not hesitate and contact me.
-        </p>
-        <button class="button">Start collaboration</button>
+      <section class="hello-content">
+        <article class="content">
+          <h1 class="my-name">Bassem Shams</h1>
+          <p class="paragraph">
+            Hello! I am a software developer! I can help you build a product,
```
Comment (thecodechaser) [inline index.html:41]: - [ ] Please place some left margins for this section to match the design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @basem909👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 5]
```diff
@@ -202,6 +280,211 @@
...
+      "project2 project2 project2 project2 . . . . . . "
+      ". . . . . . project3 project3 project3 project3"
+      "project4 project4 project4 project4 project4 .  project3 project3 project3 project3  "
+      "project4 project4 project4 project4 project4 . back2 back2 back2 back2"
+      "back3 back3 back3 . . . project5 project5 project5 project5"
+      "back3 back3 back3 project6 project6 project6 project6  . back4 back4 "
+      "back3 back3 back3 . . . . . back4 back4";
+    margin-top: 120px;
+    gap: 0;
+  }
```
Comment (phelian23) [inline styles.css]: - [ ] As requested by the previous reviewer, kindly ensure that your design matches the Figma template as much as possible. The projects grid is not properly arranged and some of the cards are stretched than they should, kindly fix this

| Your design | Figma template |
| ------------- | ------------- |
| Capture | figma |

File: styles.css [hunk 6]
```diff
@@ -455,6 +799,115 @@ label {
...
+    grid-area: user;
+  }
+
+  .email {
+    grid-area: email;
+  }
+
+  .message {
+    grid-area: message;
+  }
```
Comment (phelian23) [inline styles.css]: - [ ] As requested by the previous reviewer, kindly ensure that your design matches the Figma template as much as possible. The contact form is too wide and the `name` and `email` text field height is small compare to the template. please fix this.

| Your design | Figma template |
| ------------- | ------------- |
| Capture2 | figma2 |

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @basem909  👋,

_This is an additional review as a result of the second opinion request._

- The first requested change by @phelian23 wasnt mentioned by the first reviewer and as a result should have been OPTIONAL. but i will strongly suggest you fix the project cards some of them are a little bit over streched and the first card on hover the project should display to the right. below is a screen shot to highlight the affected sections .
- For the second requested change which is the contact form kindly adjust the width of the input filelds below is a screenshot of how centered the width of that section should look 
image

image
## NOTE : Kindly note that it is part of the learning objectives for this milestone to Understand how to parse a Figma design to create a UI. so therefore it is an important requirement that your design looks as much as possible to the figma design 🙏 

As this was the only reason that your project was not approved kindly fix the neccessary changes and it will be approved as soon as possible. 
Please ask for another review in your Dashboard and the next Code Reviewer will review it accordingly 👍 

Keep rocking 🚀!

**Review by FrederickMih — APPROVED** [review]
# Status : Approved :medal_military: :tada: 

Hi @basem909 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by basem909** [comment]
the desktop version adds media query to the site to make it responsive to different devices and browsers

**Pull request merged**